OpenRA.Game/WDist.cs

```diff
@@ -141,14 +141,14 @@ public LuaValue Equals(LuaRuntime runtime, LuaValue left, LuaValue right)
 			{
 				switch (key.ToString())
 				{
-					case "Range": return Length;
-					default: throw new LuaException("WPos does not define a member '{0}'".F(key));
+					case "Length": return Length;
```

Comment:
This is going to break any script that calls .Range.  Please keep that as an alias, and have it log a message to the game chat saying it is deprecated.